docs/builders.rst

```diff
@@ -163,3 +163,233 @@ or
 .. hint::
 
     As an alternative, you can set the config option :ref:`needs_build_needumls` to export the needumls files during each build.
+
+.. _needs_per_page_builder:
+
+needs_per_page
+--------------
+.. versionadded:: 1.4.0
+
+The **needs_per_page** builder exports all found needs with same ``docname`` into separate ``json`` file.
+If docname has slash like  ``directives/list2need``, the file will be located in folder called :ref:`needs_per_page_build_path`.
+e.g. `needs_per_page/directives/list2need.json` .
+
+Usage
++++++
+
+
+.. code-block:: bash
+
+    sphinx-build -b needs_per_page source_dir build_dir
+
+
+Format with file name: configuration.json
+++++++
+.. code-block:: python
```

Comment:
Can you move this to  a json file and present it here via `.. literalinclude::`?
Much easier to maintain.

Reply:
Maybe just remove this and use the testing snapshots.
The problem with having it "statically" in the documentation, is that it is prone to become out-of-date with the actual code